src/components/EventDetails/EventDetails.js

```diff
@@ -1138,10 +1127,10 @@ export class EventDetail extends React.Component<Props> {
       default:
         eventData = null;
     }
-    if (eventData) {
+    if (eventData && event.createdAt) {
       eventData = {
         ...eventData,
-        date: event.createdAt,
+        date: +parseDate(event.createdAt),
```

Comment:
Consider creating `parseTimestamp` function, to avoid unary `+`.
```suggestion
        date: parseTimestamp(event.createdAt),
```